index.d.ts

```diff
@@ -29,27 +40,6 @@ export type ResvgRenderOptions = {
   }
   logLevel?: 'off' | 'error' | 'warn' | 'info' | 'debug' | 'trace'
 }
-
-export type FontOptions = {
-  defaultFontSize?: number // Default: 12
-  defaultFontFamily?: string
-  serifFamily?: string
-  sansSerifFamily?: string
-  cursiveFamily?: string
-  fantasyFamily?: string
-  monospaceFamily?: string
-}
-
-export type CustomFontsOptions = {
-  fontsBuffers: Uint8Array[] // A list of raw font files to load.
-} & FontOptions
```

Comment:
There is no fontsBuffers option in index.d.ts, which is currently specific to Wasm.